app/uk/gov/hmrc/driverinspectionnotificationfrontend/models/views/InspectionDisplayGroup.scala

```diff
@@ -81,5 +85,6 @@ object InspectionDisplayGroup {
       case "7"                            => SAD
       case "8"                            => TIR
       case "13"                           => DAERA
+      case "16"                           => SnS
```

Comment:
and not a big one but if the inspectionTypeId is not one of the cases it seems to be currently throwing 5xx, can we addresses it by showing the inspectionId as is?

Reply:
scala doesn't allow using `&` in the names and better let it blow up so it alerts us with the 5xx so we can take action.. 
if we have to handle we have to create a separate DisplayGroup and message keys to handle the 5xx.. etc which is not ideal ?

Reply:
cool, i thought case value should match the reference-data inspecitonTyeps descriptions. 
and for second issue, is this not just adding the default `inspection` so it will default to inspection.heading ?

Reply:
cool, trying to see if there is a quick way of doing this. likelihood of happening this pretty rare and we already have an alert in backend to handle UKNOWN InspectionTypeIds but this scenario i'm highlighting  is not for an expected inspectionTypeId. it is in the case that it is totally ok to expect in the backend and reference-data is already aware of it but the driver ui is not. 
if the driver ui is deployed at the same time as other s&s deployments or is deployed before an s&s hold gmr is accessed the 5xx will not happen.

Reply:
happy to leave as is.

Reply:
Yeah, it's a tricky one. With the way we're currently set up where driver UI is expected to categorise inspection types, I still think it's probably better to fail fast in this instance (where we've neglected to cover a particular inspection type id which the Driver UI ought to be aware of, but isn't). 

If we ever want a catch all grouping though without explicit mappings for each type id, we could certainly add in a default.

Reply:
>  it is totally ok to expect in the backend and reference-data is already aware of it but the driver ui is not. if the driver ui is deployed after backend/ref-data or not deployed at all, don't we want to know or see an alert why driver is failing to handle rather than showing default content ?

Reply:
do we need to be alerted and that too in the form of an 5xx as we know why it is failing, the default content is to be graceful and to avoid not surfacing if the GMR has any other inspections.

Reply:
agree but what if the default content is not suitable for future inspection types and we miss to add them in the above logic ?
consider the following example and imagine having default handling everywhere for any unexpected direction type :) 
```
 override def reads(json: JsValue): JsResult[Direction] =
      json.validate[String].flatMap {
        case UK_INBOUND.value  => JsSuccess(UK_INBOUND)
        case UK_OUTBOUND.value => JsSuccess(UK_OUTBOUND)
        case GB_TO_NI.value    => JsSuccess(GB_TO_NI)
        case NI_TO_GB.value    => JsSuccess(NI_TO_GB)
        case _                 => JsError("Invalid direction type")
      }
```

Reply:
in the default case, we can say `For your other inspection types... go to location: blah`...imagine going to a location where we can't inspect so and so types ?

Reply:
no, thats alright as is. I was just hoping something like 
case "16"  => SnS
case _   => Unknown 
Unknown to mean to say "For your inspection" 
this is me simplifying it but happy to leave as is.

Reply:
yeah, we can do something like that but that `Unknown` case will be linked to a location, which means we will end up showing something like, not sure if this ok for drivers or will it confuse them, 
<img width="697" alt="Screenshot 2024-05-22 at 17 25 15" src="https://github.com/hmrc/driver-inspection-notification-frontend/assets/27204388/504df7d9-4772-4012-8ea3-d7f1b3e73f1b">